v22.1/backup-and-restore-overview.md

```diff
@@ -20,20 +20,20 @@ For an explanation of how a backup works, see [Backup Architecture](backup-archi
+------------------+--------------+-----------------+------------------------
+[Full backup](take-full-and-incremental-backups.html) | An un-replicated copy of your cluster, database, or table's data. A full backup is the base for any further backups. | All products (Enterprise license not required) | <ul><li>Customer-owned backups</li><li>Managed-service backups</li></ul>
+[Incremental backup](take-full-and-incremental-backups.html) | A copy of the changes in your data since the specified base backup (a full or full and incremental). | <ul><li>{{ site.data.products.serverless }}</li><li>{{ site.data.products.dedicated }}</li><li>{{ site.data.products.core }} with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html)</li></ul> | <ul><li>Customer-owned backups</li><li>Managed-service backups (dedicated clusters only)</li></ul>
+[Scheduled backup](manage-a-backup-schedule.html) | A schedule for periodic backups. | <ul><li>{{ site.data.products.serverless }}</li><li>{{ site.data.products.dedicated }}</li><li>{{ site.data.products.core }} with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html)</li></ul> | Customer-owned backups
+[Backups with revision history](take-backups-with-revision-history-and-restore-from-a-point-in-time.html) | A backup with revision history allows you to back up every change made within the garbage collection period leading up to and including the given timestamp. | <ul><li>{{ site.data.products.serverless }}</li><li>{{ site.data.products.dedicated }}</li><li>{{ site.data.products.core }} with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html)</li></ul> | Customer-owned backups
+[Point-in-time restore](take-backups-with-revision-history-and-restore-from-a-point-in-time.html) | A restore from an arbitrary point in time within the revision history of a backup. | <ul><li>{{ site.data.products.serverless }}</li><li>{{ site.data.products.dedicated }}</li><li>{{ site.data.products.core }} with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html)</li></ul> | Customer-owned backups
+[Encrypted backup and restore](take-and-restore-encrypted-backups.html) | An encrypted backup using a KMS or passphrase. | <ul><li>{{ site.data.products.serverless }}</li><li>{{ site.data.products.dedicated }}</li><li>{{ site.data.products.core }} with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html)</li></ul> | Customer-owned backups
+[Locality-aware backup and restore](take-and-restore-locality-aware-backups.html) | A backup where each node writes files only to the backup destination that matches the node locality configured at node startup. | <ul><li>{{ site.data.products.serverless }}</li><li>{{ site.data.products.dedicated }}</li><li>{{ site.data.products.core }} with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html)</li></ul> | Customer-owned backups                
```

Comment:
Some of these features are supported by customer-owned backups only in Dedicated (e.g. PITR, locality-aare backups, configuring your own schedule, etc.).

Reply:
@livlobo Oh ok, so locality-aware makes sense! Creating your own schedule, I have managed to set on a serverless cluster and it seems to work fine? Same with backups + revision_history and PITR?

Reply:
@livlobo OK, so this is correct then, if you're saying that on dedicated clusters only customer-owned backups support PITR, loc-aware, encrypted, schedules. These only have customer-owned backups listed. Is that what you meant?

Reply:
Yes. But maybe "backup type" should not be a different column, but rather a sub-bullet under product type? Because SH only has customer-owned backups, and Dedicated supports both, so the backup type of "customer owned, again has to be qualified with (dedicated) or (SH).